test/configs/test.config

```diff
@@ -51,24 +51,29 @@ Locations = [
     "/usr/share/polkit-1/rules.d/"
 ]
 
-[[FileDigestGroup]]
-package = "file-signature"
-bug = "boo#12345"
+[FileDigestGroup."file-signature"]
```

Comment:
Again we are missing the "package".  When you look at my suggested format I had the package encoded here. So how is this supposed to work?

Reply:
Sorry, it's maybe confusing, but `file-signature` is the name of the package. What will not work?

Reply:
Ah, sorry, I got confused by the "file-signature" label, maybe using a more descriptive name in the test would avoid this.

Reply:
Fixed that, sorry for the confusion.